projects/ngrx.io/content/guide/data/entity-actions.md

```diff
@@ -0,0 +1,269 @@
+
+The `entityName` is the default tag that appears between brackets in the formatted `type`,
+e.g., `'[Hero] NgRx Data/query-all'`.
+
+The `error` property indicates that something went wrong while processing the action. [See more below](#action-error).
+
+The `skip` property tells downstream action receivers that they should skip the usual action processing.
+This flag is usually missing and is implicitly false.
+[See more below](#action-skip).
+
+## _EntityAction_ consumers
+
+The NgRx Data library ignores the `Action.type`.
+All NgRx Data library behaviors are determined by the `entityName` and `op` properties alone.
+
+The NgRx Data `EntityReducer` redirects an action to an `EntityCollectionReducer` based on the `entityName`
+and that reducer processes the action based on the `op`.
+
+`EntityEffects` intercepts an action if its `op` is among the small set of persistence `EntityAction.op` names.
+The effect picks the right _data service_ for that action's `entityName`, then tells the service to make the appropriate HTTP request and handle the response.
+
+## Creating an _EntityAction_
+
+You can create an `EntityAction` by hand if you wish.
+The NgRx Data library considers _any action_ with an `entityName` and `op` properties to be an `EntityAction`.
+
+The `EntityActionFactory.create()` method helps you create a consistently well-formed `EntityAction` instance
+whose `type` is a string composed from the `tag` (the `entityName` by default) and the `op`.
+
+For example, the default generated `Action.type` for the operation that queries the server for all heroes is `'[Hero] NgRx Data/query-all'`.
+
+> The `EntityActionFactory.create()` method calls the factory's `formatActionType()` method
+> to produce the `Action.type` string.
+>
+> Because NgRx Data ignores the `type`, you can replace `formatActionType()` with your own method if you prefer a different format
+> or provide and inject your own `EntityActionFactory`.
+
+Note that **_each entity type has its own \_unique_ `Action` for each operation\_**, as if you had created them individually by hand.
+
+## Tagging the EntityAction
+
+A well-formed action `type` can tell the reader what changed and
+who changed it.
+
+The NgRx Data library doesn't look at the type of an `EntityAction`,
+only its `entityName` and `entityOp`.
+So you can get the same behavior from several different actions,
+each with its own informative `type`, as long as they share the same
+`entityName` and `entityOp`.
+
+The optional `tag` parameter of the `EntityActionFactory.create()` method makes
+it easy to produce meaningful _EntityActions_.
+
+You don't have to specify a tag. The `entityName` is the default tag that appears between brackets in the formatted `type`,
+e.g., `'[Hero] NgRx Data/query-all'`.
+
+Here's an example that uses the injectable `EntityActionFactory` to construct the default "query all heroes" action.
+
+```typescript
```

Comment:
If the above change is made, we'll have to verify that these code blocks also render correctly.

![image](https://user-images.githubusercontent.com/28659384/57323066-54068780-7105-11e9-8b2a-4636a5cdfc50.png)

Reply:
I think I've got them all, thanks.